MAUI/Badge-View/Badge-customization.md

```diff
@@ -344,3 +344,38 @@ Content = sfBadgeView;
 {% endtabs %}
 
 ![BadgeView Alignment](badge-customization_images/badge_alignment.png)
+
+## Visibility of badge
+
+You can show or hide the badges in the BadgeView by setting the IsVisible property in the BadgeSetting. By default, the badge will be visible.
```

Comment:
You can show or hide the badges in the .NET MAUI BadgeView by setting the `IsVisible` property in the BadgeSetting. By default, the badge will be visible.

Reply:
Updated this description.